shared/annotation.py

```diff
@@ -399,6 +397,17 @@ def __check_all_instances_available_in_new_instance_list(self):
             return False
         return True
 
+    def append_new_instance_list_hash(self, instance):
+
+        if instance.soft_delete is False:
+            self.new_instance_dict_hash[instance.hash] = instance
+            return True
+        return False
+
+    def order_new_instances_by_date(self):
```

Comment:
@PJEstrada  When do we want to call this `order_new_instances_by_date` in annotation.py?

Reply:
Hey Anthony! Thanks for all the feedback. I've done this commit to add it just before the cache check https://github.com/diffgram/diffgram/pull/225/commits/07ae4a044f6592031eff262dffd3a1736061ea36